playground/package.json

```
@@ -0,0 +1,3 @@
+{
+  "name": "automod-playground"
```

Do you mean `automd-playground`?
```suggestion
  "name": "automd-playground"
```